### PR TITLE
Various updates to the parser

### PR DESCRIPTION
**Pull request by danielgtaylor**
- Better JSON Schema reference handling (including external sources)
- Do not set resource titles
- Use `summary` instead of `operationId` for action titles
- Set a relation value based on `operationId` for actions
- Update test fixtures to support the above

cc @smizell @netmilk

### CODE REVIEW

**Comment by smizell** [comment]
Code looks good to me.

**Comment by danielgtaylor** [comment]
@smizell everything should be up to date now, with the same coverage as before. The only difference is that some covered lines were removed, so the overall coverage went down by a tiny fraction. I think this should be safe to merge.

File: src/adapter.js [hunk 1]
```diff
@@ -117,175 +103,171 @@ export function parse({minim, source}, done) {
...
 
-      // Currently, default responses are not supported in API Description format
-      const relevantResponses = _.omit(methodValue.responses, 'default');
+        const transition = new Transition();
+        resource.content.push(transition);
 
-      if (_.keys(relevantResponses).length === 0) {
-        createTransaction(minim, transition, method);
-      }
+        transition.meta.set('title', methodValue.summary || methodValue.operationId || '');
```
Comment (smizell) [inline src/adapter.js]: @netmilk You OK with falling back to `operationId` here? Or would you rather it be blank?

**Comment by smizell** [comment]
@danielgtaylor I had one question about falling back to `operationId`, but I think it is good to merge. We can figure that out later.

File: src/adapter.js (hunk 1 above)
Comment (netmilk) [inline src/adapter.js]: @smizell @danielgtaylor I'm sorry, but the title should be left empty in the refract if there's no summary in swagger. Style Guide engine other dependant tooling wouldn't know it was empty in the swagger.

**Comment by pksunkara** [comment]
:+1:

**Pull request merged**